examples/tree_1d_dgsem/elixir_shallowwater_beach.jl

```diff
@@ -0,0 +1,107 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.812)
+cfl = 0.2
+
+function initial_condition_beach(x, t, equations:: ShallowWaterEquations1D)
+  D = 1
+  δ = 0.02
+  γ = sqrt((3*δ)/(4*D))
+  xₐ = sqrt((4*D)/(3*δ)) * acosh(sqrt(20))
+
+  f = D + 40*δ * sech(γ*(8*x[1]-xₐ))^2
+
+  # steep wall
+  b = 0.01 + 99/409600 * 4^x[1]
+
+  if x[1] >= 6
+    H = b
+    v = 0
+  else
+    H = f
+    v = sqrt(equations.gravity/D) * H
+  end
+
+  H = max(H, b + equations.threshold_limiter)
```

Comment:
Same here. I would add a one line comment here as to what the `threshold_limiter` is doing and its value.